packages/app/cypress/e2e/runner/reporter.errors.cy.ts

```diff
@@ -27,10 +27,6 @@ describe('errors ui', {
   // of nested spec snapshots
   numTestsKeptInMemory: 1,
 }, () => {
-  afterEach(() => {
-    reporterSnapshot()
-  })
-
```

Comment:
You could check to see if the test or any associated runnables has an error and thus skip it here. keeping this in an `afterEach` is a good idea. There is a way to know that the test is about to fail.